tsconfig.test.json

```diff
@@ -2,7 +2,7 @@
   "extends": "./tsconfig.json",
   "compilerOptions": {
     "isolatedModules": false,
-    "types": ["jest"]
+    "types": ["jest", "vitest/globals"]
```

Comment:
Hmm this may cause trouble for us. If we're switching to `vitest` we probably want to drop the jest global types as well.

Reply:
Probably once we fully switch. Does this get rid of Jest completely or just _add_ Vitest?

Reply:
+1, this PR only adds vitest so I think we'll need both for the time being. The current plan is to move away from Jest incrementally, but it might be worth investigating the effort to do it in one fell swoop.

Reply:
We can also import vitest components directly and avoid this situation as well, it has its own pluses and minuses. Positive no implicit imports. Negative more work to migrate in general